int/int_test.go

```diff
@@ -408,7 +408,11 @@ var _ = Describe("Integrationtests", func() {
 				err = i.Client.Update(context.TODO(), &latestRouteMonitor)
 				Expect(err).NotTo(HaveOccurred())
 				_, parsedSlo := latestRouteMonitor.Spec.Slo.IsValid()
+<<<<<<< HEAD
 				err = i.RouteMonitorWaitForPrometheusRuleCorrectSLO(expectedDependentResource, parsedSlo, 20, routeMonitor.Kind)
+=======
+				err = i.WaitForPrometheusRuleCorrectSLO(expectedDependentResource, parsedSlo, 20)
+>>>>>>> 229319a (added ClusterID to templates)
```

Comment:
I think we have an unresolved merge conflict here

Reply:
Yes, I just wanted to save my changes before I went on vacation. It's very much still in Progress. Unfortunately, I lost myself a bit in refactoring, which takes just an insane amount of time although I wanted to keep it simple.